test/test_quantized.py

```diff
@@ -1502,7 +1502,7 @@ def test_qlinear(self, batch_size, input_channels, output_channels,
 
             # Observe X_fp32 and determine X_scale and X_zero_point, this should match
             # internals of dynamic linear.
-            X_scale, X_zp = _calculate_dynamic_qparams(X_fp32, torch.quint8)
+            X_scale, X_zp = _calculate_dynamic_qparams(X_fp32, torch.quint8, reduce_range=True)
```

Comment:
Please do not use reduce-range as a flag, we are planning to deprecate it. Instead use: 
``` python
          X_scale, X_zp = _calculate_dynamic_qparams(X_fp32, torch.quint8, quant_min=0, quant_max=127)
```